syft/event/parsers/parsers.go

```diff
@@ -144,30 +144,22 @@ func ParseAttestationStartedEvent(e partybus.Event) (io.Reader, progress.Progres
 
 // CLI event types
 
-func ParseCLIExit(e partybus.Event) (func() error, error) {
-	if err := checkEventType(e.Type, event.CLIExit); err != nil {
-		return nil, err
-	}
-
-	fn, ok := e.Value.(func() error)
-	if !ok {
-		return nil, newPayloadErr(e.Type, "Value", e.Value)
-	}
-
-	return fn, nil
+type UpdateCheck struct {
```

Comment:
do we want to have type definitions for these kinds of things in the parsers package? We tend to put this kind of stuff into the `monitor` package, but that name doesn't line up very well.

Reply:
This was definitely a question. Maybe a further discussion on the parsers is necessary